Tropos/Formatters/TRBearingFormatter.h

```diff
@@ -1,6 +1,6 @@
 @interface TRBearingFormatter : NSObject
 
-+ (NSString *)cardinalDirectionStringFromBearing:(CGFloat)bearing;
-+ (NSString *)abbreviatedCardinalDirectionStringFromBearing:(CGFloat)bearing;
++ (NSString *)cardinalDirectionStringFromBearing:(double)bearing;
++ (NSString *)abbreviatedCardinalDirectionStringFromBearing:(double)bearing;
```

Comment:
`CGFloat` is `double` on 64-bit architectures, and `float` on 32-bit architectures. When compiling for 32-bit, passing a `double` to this method generated a loss of precision warning. When archiving, this warning was treated as an error and failed the build.